packages/node/src/methods/app-instance/install/operation.ts

```diff
@@ -13,6 +17,9 @@ import { ERRORS } from "../../errors";
 
 export async function install(
   store: Store,
+  instructionExecutor: InstructionExecutor,
+  initiatingAddress: string,
+  respondingAddress: string,
```

Comment:
The rest of the code base uses `peerAddress` to the refer to the other Node - let's remain consistent

Reply:
This is a bit tricky because the parameters are named in the protocol specification as of the latest update by @IIIIllllIIIIllllIIIIllllIIIIllllIIIIll

Reply:
Alright, then gotta update the rest of the codebase to use `respondingAddress` then.